lib/knapsack/config/env.rb

```diff
@@ -43,6 +43,12 @@ def semaphore_current_thread
         def snap_ci_worker_index
           index_starting_from_one(ENV['SNAP_WORKER_INDEX'])
         end
+
+        def gitlab_ci_node_index
+          return unless ENV['GITLAB_CI']
```

Comment:
Should it check `CI_SERVER_VERSION` as pre `11.5` it was not supported? I guess it's not backward compatible.